src/test/renderer.test.cpp

```diff
@@ -5,26 +5,47 @@
 #include <time.h>
 
 #include "Engine.hpp"
+#include "EventSystem.hpp"
 #include "GameObject.hpp"
 #include "UI/Button.hpp"
 #include "UI/TextElement.hpp"
 
-const int WINDOW_WIDTH = 550;
-const int WINDOW_HEIGHT = 550;
-
 using namespace admirals;
+#define BIND_HANDLER(handler)                                                  \
+    std::bind(&handler, this, std::placeholders::_1, std::placeholders::_2)
```

Comment:
I feel like something like this will be commonly used, perhaps we could include it in the `EventSystem.hpp` file with a name like `BIND_METHOD_EVENT_HANDLER`?